checkDLomix.py

```diff
@@ -0,0 +1,46 @@
+#!python -m pip install -q git+https:/hub.com/wilhelm-lab/dlomix
```

Comment:
Thank you for converting the notebook to a script.

Since the code was mainly in a notebook, it needs to be adapted a bit to work as a script. Additionally, the file needs lots of formatting to be more readable.